ddmore.Rmd

```diff
@@ -0,0 +1,104 @@
+---
+title: "ddmore"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+```{r}
+library(stringr)
+library(tidyverse)
+```
+
+
+```{r}
+## one big df
+
+data_dir <- "./testdata/nmoutputs"
+file_paths <- list.files(data_dir, "\\.lst$", full.names = TRUE)
+bdf <- function(.filename){
+  df <- data.frame( filename = basename(.filename),  lines = readLines(.filename, skipNul = TRUE))
+}
+df_list <- lapply(file_paths, bdf)
+df <- do.call(rbind, df_list)
+
+```
+
+
+
+```{r}
+## one long list of lines
+lines <- unlist(lapply(file_paths, readLines, skipNul = TRUE)) %>% 
+          grep(pattern = "^[^;]", value = TRUE, useBytes = TRUE) 
+```
+
+
+```{r}
+## distinct list of lines
+dlines <- unlist(lapply(file_paths, readLines, skipNul = TRUE)) %>% 
```

Comment:
you might want to just have all the lines in a single dataframe. Eg the columns would be

file | line_contents | flag1 | flag2 | etc

where flag1/2/3 could be based on various checks you do in each line, so you might have has_dollar_est, has_dollar_table, etc. Then you can do more summaries against that, like

df %>% filter(has_dollar_est) %>% count(file) # would show number of $EST lines per file